### DISCUSSION TITLE
Get a link to the loaded wiki page

### DISCUSSION THREAD

**Discussion by androidzakzak** [rfennell/AzurePipelines#1885] [General]
Is it possible to somehow get a link to the created page on the wiki? 
I haven't found how to do this. Thank you

**Comment by rfennell** [comment]
There is no feature of the task that does this. All I can suggest is to build the page URL in a PowerShell or Bash script and use that to create a Azure DevOps pipeline variable

**Comment by androidzakzak** [comment]
Thanks
